src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -434,16 +402,25 @@ Namespace Microsoft.CodeAnalysis.Semantics
                             _semanticModel,
                             syntax,
                             type:=Nothing,
-                            constantValue:=Nothing,
-                            isImplicit:=False) ' Declaration is always explicit
+                            constantValue:=Nothing)
```

Comment:
>leftOpt?.Type [](start = 34, length = 13)

Let's handle the case of null Type, just in case this can happen with erroneous code.

Reply:
Sure, done